source/includes/_bulk_data.md.erb

```diff
@@ -20,6 +20,42 @@ The bulk data webhooks include exports of the following tables:
 
 For full information on the schema of each table, use the `/api/bulk_data/schema.json` API endpoint.
 
+### Interpreting the share_clicks table
+
+The `share_clicks` table is designed to help you understand in detail how social media sharing influences member actions.
+Every time a member clicks on a social sharing link on a petition or event page, it's recorded in the `share_clicks` table,
+and we add a unique sharing token to the campaign link they share. When another member follows the shared link and
+signs a petition, we record the `referring_share_click_id` for the new signature. Thus it is possible to trace the origin
```

Comment:
Also true for attendees?

Reply:
Not currently, unfortunately. We could/should make it so in the future, as part of petition/event parity.